planet/index.html

```diff
@@ -52,14 +63,25 @@
                 <ul class="right tabs tabs-transparent" id="localglobal">
                     <li class="tab"><a class="active" id="local-tab" href="#local"></a></li>
                     <li class="tab"><a id="global-tab" href="#global"></a></li>
+                    <li>
+                        <li>
+                            <div class="nav-content" id="searchcontainer"
+                                style="display: none;">
```

Comment:
Why inline css? Can you play around giving more specificity and achieve the same behavior with external css?